jenkins.sh

```diff
@@ -1,104 +1,14 @@
-#!/bin/bash -u
+#!/bin/bash -eu
```

Comment:
this may be a matter of taste, but I strongly prefer to see `set -e[u]` inline, so that it still happens if somebody runs the script as `bash jenkins.sh`.